main.rb

```diff
@@ -0,0 +1,158 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+    @option = ''
+    @type = ''
+    @age = ''
+    @name = ''
+    @permission = ''
+    @specialization = ''
+    @title = ''
+    @author = ''
+    @book = ''
+    @person = ''
+    @date = ''
+    @id = ''
```

Comment:
I strongly suggest you use arrays to store data else your code base might break along the way 👍🏾

Reply:
Hello, thank you for the suggestion, but shouldn't this be an optional change as this was not stated in the requirements. thanks.